src/services/me/index.js

```diff
@@ -1,5 +1,6 @@
 const { Forbidden, GeneralError } = require('@feathersjs/errors');
 const logger = require('../../logger');
+const { userModel } = require('../../services/user/model');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Useless path segments for "../../services/user/model", should be "../user/model"](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5229889)